packages/react/src/components/Tabs/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+class Tabs extends Component {
```

Comment:
The same comment about hooks, there is no reason to ship code we can quickly improve
(Let me know if I can help)